### PR TITLE
UI minor changes

### PR DESCRIPTION
**Pull request by comzeradd**

### CODE REVIEW

**Comment by comzeradd** [comment]
@akatsoulas r?

**Comment by akatsoulas** [comment]
WhiteNoise isn't very happy due to some missing files.

File: feedthefox/base/templates/base.jinja [hunk 1]
```diff
@@ -31,14 +32,16 @@
...
-              <li><a href="https://support.mozilla.org/products/firefox-os" target="_blank">Support</a></li>
-              <li><a href="https://developer.mozilla.org/Firefox_OS" target="_blank">Documentation</a></li>
               <li><a href="{{ url('dashboard') }}">Dashboard</a></li>
+              <li><a href="#">
+                <span class="glyphicon glyphicon-user" aria-hidden="true"></span> Connect
+              </a></li>
             </ul>
           </div>
+
           <div id="tabzilla" class="hidden-xs hidden-sm">
```
Comment (akatsoulas) [inline feedthefox/base/templates/base.jinja]: Personally, I think that  we can remove `hidden-sm` class. Thoughts?

**Comment by akatsoulas** [comment]
r+wc :+1:

**Comment by comzeradd** [comment]
@akatsoulas PR update. Added the missing svg files.

**Pull request merged**